Dynatrace/dynatrace-operator
# Remove yamls that are generated in `manifests` build target

# Description
Removed files from the repo:
- `config/deploy/kubernetes/kubernetes-all.yaml`
- `config/deploy/openshift/openshift-all.yaml`

because they are generated in `manifests` build target.

## How can this be tested?
1. `make manifests`
2. verify if files are generated `find ./config -name "*-all.yaml"`

## Checklist
